add_package.yml

```diff
@@ -0,0 +1,28 @@
+---
+- hosts: packages
+  serial: 1
+  gather_facts: no
+  vars:
+    gem_name: "{{ inventory_hostname.replace('tfm-ror51-rubygem-', '') }}"
+  roles:
+    - git-annex-setup
+  tasks:
+    - name: 'Create package directory'
+      file:
+        state: directory
+        path: "packages/{{ inventory_hostname }}"
+
+    - name: 'Fetch gem'
+      command: "gem fetch {{ gem_name }} --version {{ version }}"
```

Comment:
This could use a ```creates``` in case you want to rerun the spec generation.

Reply:
I think that we do in the case of adding a package. At least, the use case of modifying the template to regenerate specs shortens the workflow.